README.MD

```diff
@@ -1,46 +1,102 @@
-## Synopsis
+# SplashKit Manager (SKM)
 
-SplashKit Manager or SKM is the CLI/App tool for installing and managing SplashKit, as well as creating, building, and running your very own SplashKit projects!
+<p align="left">
+    <img width="150px" src="https://github.com/thoth-tech/.github/blob/main/images/splashkit.png"/>
+</p>
 
-## Use Example
+### [Splashkit](https://github.com/splashkit/skm) Stats
 
-* ```cd /code/projects```
-* ```mkdir helloworld```
-* ```cd helloworld```
-* ```skm new c++```
-* ```skm clang++ *.cpp -o HelloWorld```
-* ```./HelloWorld```
+[![GitHub contributors](https://img.shields.io/github/contributors/splashkit/skm?label=Contributors&color=F5A623)](https://github.com/splashkit/skm/graphs/contributors)
+[![GitHub issues](https://img.shields.io/github/issues/splashkit/skm?label=Issues&color=F5A623)](https://github.com/splashkit/skm/issues)
+[![GitHub pull requests](https://img.shields.io/github/issues-pr/splashkit/skm?label=Pull%20Requests&color=F5A623)](https://github.com/splashkit/skm/pulls)
+[![Forks](https://img.shields.io/github/forks/splashkit/skm?label=Forks&color=F5A623)](https://github.com/splashkit/skm/network/members)
+[![Stars](https://img.shields.io/github/stars/splashkit/skm?label=Stars&color=F5A623)](https://github.com/splashkit/skm/stargazers)
 
-## Motivation
+### [Thoth Tech](https://github.com/thoth-tech/skm) Stats
 
-SKM is designed to simplify the process of compiling and running programs that use the SplashKit library.
+[![GitHub contributors](https://img.shields.io/github/contributors/thoth-tech/skm?label=Contributors&color=F5A623)](https://github.com/thoth-tech/skm/graphs/contributors)
+[![GitHub issues](https://img.shields.io/github/issues/thoth-tech/skm?label=Issues&color=F5A623)](https://github.com/thoth-tech/skm/issues)
+[![GitHub pull requests](https://img.shields.io/github/issues-pr/thoth-tech/skm?label=Pull%20Requests&color=F5A623)](https://github.com/thoth-tech/skm/pulls)
+[![Forks](https://img.shields.io/github/forks/thoth-tech/skm?label=Forks&color=F5A623)](https://github.com/thoth-tech/skm/network/members)
+[![Stars](https://img.shields.io/github/stars/thoth-tech/skm?label=Stars&color=F5A623)](https://github.com/thoth-tech/skm/stargazers)
 
-## Installation
+**SplashKit Manager** (SKM) is a command-line interface (CLI) and app tool designed for installing, managing, and building projects with the SplashKit SDK. With SKM, you can easily create, compile, and run SplashKit projects, making it a streamlined tool for beginners and experienced developers alike.
 
-Refer to the SplashKit [installation guide](http://www.splashkit.io/articles/installation/).
+## Usage Example
```

Comment:
I would suggest updating this to be "Using SplashKit" or similar, to avoid confusion with the Usage Examples on the website.